### PR TITLE
Add note about setting fractional gates flag

### LINKED ISSUE #1527: update runtime options page(s) to include fractional gates flag

### PR DESCRIPTION
**Pull request by kaelynj**
Closes #1527 

~~Currently blocked by #1734 which will need to be merged before CI passes.~~

Utilizing fractional gates is a flag that is set when requesting a backend from a `QiskitRuntimeService()` instance and there didn't appear to be a clear page for a flag to be explained. I ended up adding it as an admonition to the "get started with primitives" page.

### CODE REVIEW

File: docs/guides/get-started-with-primitives.mdx [hunk 1]
```diff
@@ -23,6 +23,15 @@ service = QiskitRuntimeService(channel="ibm_cloud", channel_strategy="q-ctrl")
     While this documentation uses the primitives from Qiskit Runtime, which allow you to use IBM&reg; backends, the primitives can be run on any provider by using the [backend primitives](#backend) instead.  Additionally, you can use the *reference* primitives to run on a local statevector simulator.  See [Exact simulation with Qiskit primitives](simulate-with-qiskit-sdk-primitives) for details.
 </Admonition>
 
+
+<Admonition type="note" title="Using Fractional Gates">
+    If you wish to use the newly supported [fractional gates](fractional-gates), you will need to set `use_fractional_gates=True` when requesting a backend from a `QiskitRuntimeService` instance. For example:
```
Comment (Eric-Arellano) [inline docs/guides/get-started-with-primitives.mdx:28]:
```suggestion
    If you wish to use the newly supported [fractional gates](./fractional-gates), you will need to set `use_fractional_gates=True` when requesting a backend from a `QiskitRuntimeService` instance. For example:
```

**Comment by jyu00** [comment]
Note that the rollout of fractional gate has been delayed until August. So perhaps we want to hold on to this update until a later time.

**Comment by kaelynj** [comment]
> Note that the rollout of fractional gate has been delayed to August. So perhaps we want to hold on to this update until a later time.

Yeah we'll put this on hold until the fractional gates are released.

File: docs/guides/get-started-with-primitives.mdx [hunk 2]
```diff
@@ -23,6 +23,15 @@ service = QiskitRuntimeService(channel="ibm_cloud", channel_strategy="q-ctrl")
     While this documentation uses the primitives from Qiskit Runtime, which allow you to use IBM&reg; backends, the primitives can be run on any provider by using the [backend primitives](#backend) instead.  Additionally, you can use the *reference* primitives to run on a local statevector simulator.  See [Exact simulation with Qiskit primitives](simulate-with-qiskit-sdk-primitives) for details.
 </Admonition>
 
+
+<Admonition type="note" title="Using Fractional Gates">
+    If you wish to use the newly supported [fractional gates](./fractional-gates), you will need to set `use_fractional_gates=True` when requesting a backend from a `QiskitRuntimeService` instance. For example:
```
Comment (nkanazawa1989) [inline docs/guides/get-started-with-primitives.mdx:28]: It's worth saying this behavior might change in the future.

**Review by beckykd — APPROVED** [review]

**Pull request merged**